docs: update privacy policy and conditions of use (#5129)
simplex-chat/simplex-chat: PRIVACY.md

permalink: /privacy/index.html
---

# SimpleX Chat Operators Privacy Policy and Conditions of Use

## Summary

[Introduction](#introduction) and [General principles](#general-principles) cover SimpleX Chat network design, the network operators, and the general principles for privacy and security that SimpleX network provides.

[Privacy policy](#privacy-policy) covers:

> - data stored only on your device - [your profiles](#user-profiles), delivered [messages and files](#messages-and-files). This information can be transferred to another device, and you are responsible for its preservation - if you delete the app it will be lost.

Reviewer: ```suggestion
- data stored only on your device - [your profiles](#user-profiles), delivered [messages and files](#messages-and-files). This information can be transferred to another device by user, and you are responsible for its preservation - if you delete the app it will be lost.
```
or "user can transfer"

- [private message delivery](#private-message-delivery) that protects your IP address and contacts from the destination servers.

> - information is stored on the servers ([undelivered messages and files](#storage-of-messages-and-files-on-the-servers)). It also explains how [connections between users](#connections-with-other-users) work.

Reviewer: ```suggestion
- information stored on the servers ([undelivered messages and files](#storage-of-messages-and-files-on-the-servers)). It also explains how [connections between users](#connections-with-other-users) work.
```

- [iOS push notifications](#ios-push-notifications) privacy limitations.
- [User support](#user-support), [SimpleX directory](#simplex-directory) and [any other data](#another-information-stored-on-the-servers) that may be stored on the servers.
- [Preset server operators](#preset-server-operators) and the [information they can share](#information-preset-operators-may-share)
- [Source code license](#source-code-license) and [updates to this document](#updates).

> [Conditions of Use of Software and Infrastructure](#conditions-of-use-of-software-and-infrastructure) cover the conditions you need to accept to use SimpleX Chat Applications and the relay servers of preset operators. Their purpose is to protect preset server operators and other users from any malicious actions of other users.

Reviewer: ```suggestion
[Conditions of Use of Software and Infrastructure](#conditions-of-use-of-software-and-infrastructure) cover the conditions you need to accept to use SimpleX Chat Applications and the relay servers of preset operators. Their purpose is to protect preset server operators and users from any malicious actions of other users.
```

> This summary is provided for convenience only, it is not part of the Privacy Policy and Conditions of Use.

Reviewer: probably better to use same name everywhere? or give short names in previous paragraph: `Conditions of Use of Software and Infrastructure (further, Conditions of Use [or Conditions]) `

Double ratchet algorithm has such important properties as [forward secrecy](/docs/GLOSSARY.md#forward-secrecy), sender [repudiation](/docs/GLOSSARY.md#) and break-in recovery (also known as [post-compromise security](/docs/GLOSSARY.md#post-compromise-security)).

If you believe that any part of this document is not aligned with SimpleX network mission or values, please raise it via [email](mailto:chat@simplex.chat) or [chat](https://simplex.chat/contact#/?v=1&smp=smp%3A%2F%2FPQUV2eL0t7OStZOoAsPEV2QYWt4-xilbakvGUGOItUo%3D%40smp6.simplex.im%2FK1rslx-m5bpXVIdMZg9NLUZ_8JBm8xTt%23%2F%3Fv%3D1%26dh%3DMCowBQYDK2VuAyEALDeVe-sG8mRY22LsXlPgiwTNs9dbiLrNuA7f3ZMAJ2w%253D%26srv%3Dbylepyau3ty4czmn77q4fglvperknl4bi2eb2fdy2bh4jxtf32kf73yd.onion).

## Privacy Policy

### General principles

> SimpleX network software uses the best industry practices for security and encryption to provide client and server software for secure [end-to-end encrypted](/docs/GLOSSARY.md#end-to-end-encryption) messaging via private connections. This encryption cannot be compromised by the relays servers, even if they are modified or compromised, via [man-in-the-middle attack](/docs/GLOSSARY.md#man-in-the-middle-attack), unlike most other communication platforms, services and networks.

Reviewer: "cannot be compromised" - conjecture, why put it in privacy policy. That it is believed to be not possible to be compromised now doesn't mean it cannot be ever.

Your message history is stored only on your own device and the devices of your contacts. While the recipients' devices are offline, messaging relay servers temporarily store end-to-end encrypted messages – you can configure which relay servers are used to receive the messages from the new contacts, and you can manually change them for the existing contacts too.

#### Private message delivery 

You do not have control over which servers are used to send messages to your contacts - they are chosen by them. To send messages your client by default uses configured servers to forward messages to the destination servers, thus protecting your IP address from the servers chosen by your contacts.

> In case you use preset servers of more than one operator, the app will prefer to use the server of another operator to forward messages than the operator of the destination server, thus preventing destination server to correlate messages as belonging to one client.

Reviewer: ```suggestion
In case you use preset servers of more than one operator, the app will prefer to use a server of an operator different from the operator of the destination server to forward messages, thus preventing destination server to correlate messages as belonging to one client.
```

You can additionally use VPN or some overlay network (e.g., Tor) to hide your IP address from the servers chosen by you.

*Please note*: the clients allow changing configuration to connect to the destination servers directly. It is not recommended - if you make such change, your IP address will be visible to the destination servers.

#### Storage of messages and files on the servers

> The messages are removed from the used relay servers as soon as file they were stored in is fully delivered and saving messages switches to another file, as long as these servers use unmodified published code. Undelivered messages are also marked as delivered after the time that is configured in the messaging servers you use (21 days for preset messaging servers).

Reviewer: ```suggestion
The messages are removed from the used relay servers as soon as all messages of file they were stored in are fully delivered and saving messages switches to another file, as long as these servers use unmodified published code. Undelivered messages are also marked as delivered after the time that is configured in the messaging servers you use (21 days for preset messaging servers).
```

**Damage to SimpleX Chat Ltd and Preset Server Operators**. You must not (or assist others to) access, use, modify, distribute, transfer, or exploit SimpleX Chat Applications in unauthorized manners, or in ways that harm Software users, SimpleX Chat Ltd, other preset server operators, their Infrastructure, or any other systems. For example, you must not 1) access preset operators' Infrastructure or systems without authorization, in any way other than by using the Software; 2) disrupt the integrity or performance of preset operators' Infrastructure; 3) collect information about the users in any manner; or 4) sell, rent, or charge for preset operators' Infrastructure. This does not prohibit you from providing your own Infrastructure to others, whether free or for a fee, as long as you do not violate these Conditions and AGPLv3 license, including the requirement to publish any modifications of the relay server software.

> **Keeping your data secure**. SimpleX Chat is the first communication software that aims to be 100% private by design - server software neither has the ability to access your messages, nor it has information about who you communicate with. That means that you are solely responsible for keeping your device, your user profile and any data safe and secure. If you lose your phone or remove the Software from the device, you will not be able to recover the lost data, unless you made a back up. To protect the data you need to make regular backups, as using old backups may disrupt your communication with some of the contacts. SimpleX Chat Ltd and other preset responsibility have no responsibility for the data loss.

Reviewer: ```suggestion
**Keeping your data secure**. SimpleX Chat is the first communication software that aims to be 100% private by design - server software neither has the ability to access your messages, nor it has information about who you communicate with. That means that you are solely responsible for keeping your device, your user profile and any data safe and secure. If you lose your phone or remove the Software from the device, you will not be able to recover the lost data, unless you made a back up. To protect the data you need to make regular backups, as using old backups may disrupt your communication with some of the contacts. SimpleX Chat Ltd and other preset server operators responsibility have no responsibility for the data loss.
```
